src/Tasks/GenerateResource.cs

```diff
@@ -1660,16 +1681,23 @@ private void GetStronglyTypedResourceToProcess(ref List<ITaskItem> inputsToProce
             DateTime sourceTime = NativeMethodsShared.GetLastWriteFileUtcTime(Sources[0].ItemSpec);
             DateTime outputTime = NativeMethodsShared.GetLastWriteFileUtcTime(StronglyTypedFileName);
 
-            if (sourceTime == DateTime.MinValue || outputTime == DateTime.MinValue)
+            if (sourceTime == DateTime.MinValue)
             {
                 // Source file is missing - force a build, so that resource generation
-                // will produce a nice error message; or output file is missing,
-                // need to build it
+                // will produce a nice error message
+                Log.LogMessageFromResources(MessageImportance.Low, "GenerateResource.InputDoesntExist", Sources[0].ItemSpec);
```

Comment:
Were you able to hit this codepath? My naive attempt to do so failed, because an earlier error fired and we never got to here.

Reply:
I can't remember. Feel free to remove if you like, or I can?